test/sequential/test-resource-usage.js

```diff
@@ -0,0 +1,23 @@
+var assert = require('assert');
+var common = require('../common');
```

Comment:
The linter is probably going to complain that this should have `'use strict'` at the top and that the imports should be `const`.